src/components/target-pane/target-pane.jsx

```diff
@@ -113,4 +132,8 @@ TargetPane.propTypes = {
     vm: React.PropTypes.instanceOf(VM)
 };
 
+TargetPane.defaultProps = {
+    mediaLibrary: new MediaLibrary()
```

Comment:
What's the lifetime for this? Specifically, does it survive this component or the parent component re-rendering?